### PR TITLE
show the error output from validatorLogs func in lov viewer

### LINKED ISSUE #41: Need to display error when Docker is not detected
As long as we're leaning on Docker, we need to communicate to the user if it's not available.

### PR DESCRIPTION
**Pull request by SvenDowideit**
closes #41

### CODE REVIEW

File: src/main/validator.ts [hunk 1]
```diff
@@ -48,13 +57,17 @@ const validatorLogs = async (msg: ValidatorLogsRequest) => {
   const MAX_DISPLAY_LINES = 30;
 
   if (net !== Net.Localhost) {
-    return '';
+    return `Cannot show validator container output from ${net}`;
```
Comment (nathanleclaire) [inline src/main/validator.ts:60]: IIRC this should be a proper `throw new Error(`Cannot show validator container output from ${net}`)` and lint might well complain about it

**Review by nathanleclaire — CHANGES_REQUESTED** [review]

**Comment by nathanleclaire** [comment]
very nice! appreciate this one as it's been a thorn in my side for ages.

have a couple lint issues build is complaining about, but once those are fixed, lgtm

**Comment by SvenDowideit** [comment]
I'm going to claim the `console.log` thing is because https://github.com/workbenchapp/solana-workbench/pull/94 isn't merged yet :)

wrt the `throw Error` - exceptions are ... for exception cases, not for normal, detected results - and this is not exceptional.

**Comment by nathanleclaire** [comment]
> wrt the throw Error - exceptions are ... for exception cases, not for normal, detected results - and this is not exceptional.

either/or, if it's ok with lint, fine by me, just seem to recall my VSCode complaining about it

**Comment by nathanleclaire** [comment]
now that #94 is in business, lgtm once log lines are updated

**Review by nathanleclaire — APPROVED** [review]

**Comment by SvenDowideit** [comment]
yeah, that linter error was weird - mixing `return string` and `return { output }` - for now i'm going with just returning a string, and showing that in the UI - atm some point, I hope to upgrade the ipc code to something typed - that thunking makes me cringe like its 1996

**Pull request merged**